pkg/operator/handler.go

```diff
@@ -52,6 +52,7 @@ func NewHandler(namespace string) (sdk.Handler, error) {
 	p.Healthz.TimeoutSeconds = 5
 
 	p.Service.Name = "image-registry"
+	p.ImageConfig.Name = "cluster"
```

Comment:
@deads2k @juanvallejo  relevant to your "observe the internal registry hostname" PR.  

this PR adds the logic to the image-registry operator to publish the config object, named "cluster" (per @deads2k's "canonical" comment on the api type)

Reply:
@bparees thanks, I'll go ahead and update https://github.com/openshift/cluster-openshift-apiserver-operator/pull/41 accordingly